### PR TITLE
Add proposed tax estimate feature

### PR DESCRIPTION
**Pull request by timwis**
Pending OPA/Finance approval. Merge to deploy once approved. I'm out of the office on Friday 4/20.

/cc @rbrtmrtn @tswanson

### CODE REVIEW

**Comment by rmx90210** [comment]
This looks nice and is working well for me. My only suggestion would be to remove the "Estimate" button, since after you click it there's no way to get it to come back. Other than that, I think this is good to go, if we ultimately decide to put this into Property.

**Comment by timwis** [comment]
Thanks @rbrtmrtn the reason that was added was to decrease the likelihood someone might assume the number is their 2019 taxes -- it's just an estimate of the taxes based on the two rates being discussed, and doesn't account for certain tax assistance programs like OOPA (as far as I know). By making them click the "estimate" button, it (hopefully) increases the likelihood that they're aware it's an estimate. And it's a carry-over from the avi calculator.

**Comment by rmx90210** [comment]
Thanks @timwis, sounds good. We were seeing a few issues with how the homestead exemption was getting applied so I pushed a few changes. I don't think I can deploy to your Surge site, so I'll create another demo and send it out.

**Comment by tswanson** [comment]
Don't  push any updates until we get signoff on the changes that have been document

**Comment by rmx90210** [comment]
@tswanson Sure thing, we'll pick up next week.

File: js/views/property.js [hunk 1]
```diff
@@ -378,6 +378,108 @@ app.views.property = function (accountNumber) {
...
+      $(document).foundation();
+
+      var existingHomesteadValue = opa.homestead_exemption,
+          rateVal = app.hooks.taxEstimateRate.val(),
+          displayHomesteadValue;
+
+      // if they already have a homestead exemption, use that value
+      if (existingHomesteadValue > 0) {
+        if (rateVal === 'proposed') {
+          displayHomesteadValue = 45000;
```
Comment (timwis) [inline js/views/property.js]: Shouldn't this be `Math.max(0, existingHomesteadValue / 30000 * 45000)`?

Comment (rmx90210) [reply js/views/property.js]: Good catch, yes -- it should reflect the proportionate, proposed homestead. I'll push a change.

**Review by rmx90210 — COMMENTED** [review]
@timwis I updated this so it shows the proportionate amount. I don't think the `Math.max()` was necessary -- the existing homestead would have to be negative to end up with a negative proposed homestead, and I don't think there's anything like that in the data.

**Pull request merged**